app/views/result/index.html.erb

```diff
@@ -68,85 +76,120 @@
         test in and a counter to increment within each of the following
         loops                           %>
     <% a_list = {} %>
+    <% a_list_titles = [] %>
     <% a_counter = 0 %>
     <% b_counter = 0 %>
-    <div class="AB_search_wrapper_left">
-      <h2 class="results-header">A Results</h2>
-
-      <%# Adds the A test parameter to the URL  %>
-      <% uri = URI("#{url}&ab_tests=#{params["which_test"]}:A") %>
-
-      <%# Gets a result from the search API based on the URL we made %>
-      <% response = Net::HTTP.get(uri) %>
-      <% findings = JSON.parse(response) %>
-
-      <%# Loops over the 'results' section of our result %>
-      <% findings["results"].each do |each_bit| %>
 
-        <%# increments counter and creates hash entry based on link and counter %>
-        <% a_counter += 1 %>
-        <% a_list["#{each_bit["link"]}"] = a_counter %>
+    <%# <h2 class="results-header">A Results</h2> %>
+    <%# Adds the A test parameter to the URL  %>
+    <% uri = URI("#{url}&ab_tests=#{params["which_test"]}:A") %>
+    <%# Gets a result from the search API based on the URL we made %>
+    <% response = Net::HTTP.get(uri) %>
+    <% findings_left = JSON.parse(response) %>
+    <%# Loops over the 'results' section of our result %>
+    <% findings_left["results"].each do |each_bit| %>
+      <%# increments counter and creates hash entry based on link and counter %>
+      <% a_counter += 1 %>
+      <% a_list["#{each_bit["link"]}"] = a_counter %>
+      <% a_list_titles << each_bit["title"] %>
+    <% end %>
+    <% uri = URI("#{url}&ab_tests=#{params["which_test"]}:B") %>
+    <% response = Net::HTTP.get(uri) %>
+    <% findings_right = JSON.parse(response) %>
+
+    <table class="results-table" >
+      <tr class="table-row" class="table-row">
+        <th scope="col"><h3 class="heading-27" >A Results</h3></th>
+        <th scope="col"><h3 class="heading-27" >B Results</h3></th>
+        <th scope="col"><h3 class="heading-27" >Difference</h3></th>
+      </tr>
+      <% b_counter = 0 %>
+      <% findings_right["results"].each do |each_bit| %>
+        <tr class="table-row">
+          <% if a_list["#{each_bit["link"]}"] != nil %>
```

Comment:
as `nil` is falsey and any value that is not `nil` can be assumed to be true in this case you can simplify this to:
```<% if a_list["#{each_bit["link"]}"] %>```